src/io/flutter/actions/OpenInAndroidStudioAction.java

```diff
@@ -189,20 +190,24 @@ private static String findAndroidStudio(@Nullable Project project) {
       return null;
     }
 
+    // TODO: If we're running in Android Studio (FlutterUtils.isAndroidStudio()), we should return ourself as the location.
```

Comment:
If running in Android Studio we never get here. It uses a subclass of this class to open a new project window rather than trying to launch itself.

Reply:
gotcha - will delete the comment